ansible/roles/ocp-infra-osp-fip/README.md

```diff
@@ -0,0 +1,42 @@
+AWS-Broker
```

Comment:
LOL.

Reply:
Guess I should fix the README.md to actually have validate information... oops